content/_data/changelogs/lts.yml

```diff
@@ -3835,6 +3835,19 @@
     message: |-
       Fix buttons that linger too long after closing modal (regression in 2.233 and 2.235.1).
 
+- version: "2.235.3"
+  date: 2020-07-27
+  changes:
+  - type: major rfe
+    category: major rfe
+    message: |-
+      Update Debian and Red Hat repository signing keys for the Jenkins long term support repositories.
```

Comment:
it worth mentioning that other artifact signing also changes (WAR/MSI). Signing there does not require extra update steps by default, but IMHO this is still a major change. Could be a separate changelog entry

Reply:
Added in 2959af22e0157f1d5ea0cd1a10b873ff3cd56c29